x-pack/plugins/alerting/server/rules_client/methods/bulk_edit.ts

```diff
@@ -738,6 +741,37 @@ function validateScheduleInterval(
   }
 }
 
+/**
+ * Validate that updated schedule interval is not longer than any of the existing action frequencies
+ * @param schedule Schedule interval that user tries to set
+ * @param actions Rule actions
+ */
+function validateScheduleOperation(
+  schedule: RawRule['schedule'],
+  actions: RawRule['actions']
+): void {
+  const scheduleInterval = parseDuration(schedule.interval);
+  const actionsWithInvalidThrottles = [];
+
+  for (const action of actions) {
+    // check for actions throttled shorter than the rule schedule
+    if (
+      action.frequency?.notifyWhen === RuleNotifyWhen.THROTTLE &&
+      parseDuration(action.frequency.throttle!) < scheduleInterval
+    ) {
+      actionsWithInvalidThrottles.push(action);
+    }
+  }
+
+  if (actionsWithInvalidThrottles.length > 0) {
+    throw Error(
+      `Error updating rule: the interval is longer than the action frequencies: ${actionsWithInvalidThrottles
```

Comment:
Can we just add the ruleId and/or the ruleName in the error?

Reply:
ah yeah, that would be more useful!